test/help.cc

```diff
@@ -17,14 +17,17 @@ TEST(FlagsTest, Help) {
   int argc = arguments.size();
   const char** argv = arguments.data();
 
+  const std::string match_regex =
```

Comment:
The variable name `match_regex` seems redundant, let's just:
```suggestion
  const std::string regex =
```

Here and everywhere else please!